docs/rest_api_2.rst

```diff
@@ -0,0 +1,4156 @@
+Can be retrieved as follows:
+
+1. POST a request to the following API endpoint
+
+.. code-block:: json 
+
+   https://<APIC-URL>/api/aaaLogin.xml
+
+2. Body for the POST request
+
+.. code-block:: json
+
+   <aaaUser name="apic-username" pwd="apic-password"/>
+
+3. From the result of the POST, save the token returned
+
+4. In the “Headers” section of any further REST API requests, add a key-value pair. Key name is “DevCookie”, and its value should be the token obtained in the previous step
+
+.. code-block:: json
+   
+   Key         Value        
+   DevCookie : < token value > 
+   
+
+Header: Content-Type
+````````````````````
+.. code-block:: json
+   
+   Key           Value           
+   Content-Type: application/json
+   
+Input Parameters
+----------------
+
+There are multiple F5 ACI ServiceCenter APIs which require input parameters to be retrieved for the APIC Logical Device. These input parameters can be retrieved from APIC object browser known as visore.html
+
+APIC managed objects can be accessed at https://<APIC_IP>/visore.html.
+
+Below is a table of such parameters and steps on how to retrieve them.
+These parameters will be required for L2-L3 Stitching tab.
+
++-------------+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Input Key   | Input Value                                                                                                                    | Input Value Example                                                                                                                                     |
++=============+================================================================================================================================+=========================================================================================================================================================+
+| ldev        | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device>                                                      |     uni/tn-Sample\_2/lDevVip-f5-gs                                                                                                                      |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the <Your L4-L7 Device> and click on “Open in Object Store Browser”                                          |                                                                                                                                                         |
```

Comment:
remove both "on" and replace quotes with ** **, ie Right click the <your L4..., and click **Open in Object...

Reply:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)